src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    private static final Figure[] FIGURES = new Figure[] {new Circle(), new Triangle(),
+            new Square(), new Rectangle(), new IsoscelesTrapezoid()};
+
+    public static Figure randomFigure() {
```

Comment:
Let's use a more informative method naming
```suggestion
    public static Figure getRandomFigure() {
```

Reply:
done